### ISSUE TITLE
repeated error on jobs page

### ISSUE THREAD

**Issue by duncanrand** [ic-hep/pdm#420]
Error message:

DataTables warning: table id=jobs_table - Cannot reinitialise DataTable. For more information about this error, please see http://datatables.net/tn/3

**Comment by alexanderrichards** [comment]
yes, this is already in the process of being fixed. I'll add a PR soon.
